thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/pinot/SqlUtils.java

```diff
@@ -393,7 +398,7 @@ static String quote(String value) {
     String quoteChar = "";
     if (!StringUtils.isNumeric(value)) {
       quoteChar = "\"";
-      if (value.contains(quoteChar)) {
+      if (value.contains(quoteChar) || StringUtils.isEmpty(value)) {
```

Comment:
StringUtils.isEmpty also checks for null. Better to run this check first `StringUtils.isEmpty(value) || value.contains(quoteChar)`